README.md

```diff
@@ -16,8 +16,13 @@ To use this task a token is required. Don't have one? We're here to help.
   inputs:
     url:
     token: ${{ secrets.AUTOMAGICALLY_TOKEN }}
+    testTargetId: <your testTargetId that you also get from us>
 ```
 
+## Change Log
+
+- 2023-07-23: Added requirement for setting `testTargetId` to enable v2 API
```

Comment:
you will also need to bump the versions in both the task.json AND vss-extension.json for azure

Reply:
Done, and I added testTargetId to task.json.
I made it version 2.0.0 due to breaking change